charts/opentelemetry-operator/templates/deployment.yaml

```diff
@@ -58,10 +58,12 @@ spec:
             initialDelaySeconds: 5
             periodSeconds: 10
           resources: {{ toYaml .Values.manager.resources | nindent 12 }}
+          {{- if and .Values.admissionWebhooks.enabled .Values.admissionWebhooks.certManager.enabled }}
```

Comment:
Why not `{{- if .Values.admissionWebhooks.enabled }}`? Does setting `admissionWebhooks=false` and `admissionWebhooks.certManager=true` has any meaning/effect?

Reply:
nvm. I misunderstood this